craftmaster.ini

```
@@ -38,8 +38,8 @@ Packager/CacheDir = ${Variables:Root}\cache
 [BlueprintSettings]
 nextcloud-client.buildTests = True
 binary/mysql.useMariaDB = False
-libs/qt6.version = 6.8.3
-craft/craft-blueprints-kde.revision = stable-3.17
+libs/qt6.version = 6.9.1
```

Please use the next patch version because it resolves a fatal build error on macOS 26 Tahoe.
```suggestion
libs/qt6.version = 6.9.2
```
https://codereview.qt-project.org/c/qt/qtbase/+/652022/3